### ISSUE TITLE
Unclaiming a merged region by breaking blocks gives back the first block you break

### ISSUE THREAD

**Issue by espidev** [espidev/ProtectionStones#131]
From SpigotMC (MCWaffles): 
```
We have 4 sizes, 2 that go from bedrock to sky and 2 that are smaller and are cubes. When merging the smaller 2 with any other block (with each other or with a bigger one) whichever block you break first will also be the block you get back when you break the second. So if you merge a small 9x9x9 cube with a 128x(bedrock-sky)x128 and you break the 128 first and then the 9x, and you will have 2x 128x in your inventory (the other way around you would have two 9x in your inventory).

This happens both on 1.14.4 and 1.15.1 for me.


i just tried to do it one more time to get the info you asked for and it seems to only happen when the smaller claim is completely inside of the bigger claim.

In the image the first is when the claims are overlapping and the second is when they are not, the first i got 2 the same blocks back and the second i got the right blocks back.

In your config it reads like you can use it if you don't mind the regions going from bedrock to sky, which is why we gave perms to merge only to higher ranked players.
ps.png
```

**Comment by espidev** [comment]
evidence: https://www.youtube.com/watch?v=1KtodTdTeAk

**Comment by espidev** [comment]
Fixed, will be in 2.6.6
